File: app/src/main/java/at/shockbytes/dante/util/books/BookSuggestion.java
package at.shockbytes.dante.util.books;

import java.util.List;

/**
 * @author Martin Macheiner
 *         Date: 11.09.2017.
 */

public class BookSuggestion {

    private Book mainSuggestion;

    private List<Book> otherSuggestions;

    public BookSuggestion(Book mainSuggestion, List<Book> otherSuggestions) {
        this.mainSuggestion = mainSuggestion;
        this.otherSuggestions = otherSuggestions;
    }

    public Book getMainSuggestion() {
        return mainSuggestion;
    }

    public List<Book> getOtherSuggestions() {
        return otherSuggestions;
    }

    public boolean hasSuggestions() {
        return mainSuggestion != null;
    }

}

File: app/src/main/java/at/shockbytes/dante/util/books/BookSuggestion.kt
package at.shockbytes.dante.util.books

/**
 * @author Martin Macheiner
 * Date: 11.09.2017.
 */

class BookSuggestion(val mainSuggestion: Book?, val otherSuggestions: List<Book>) {

    val hasSuggestions: Boolean
        get () = mainSuggestion != null

}
